sphinx/writers/latex.py

```diff
@@ -552,6 +575,15 @@ def __init__(self, document, builder):
                            builder.config.language)
 
         # set up multilingual module...
+        if self.elements['latex_engine'] == 'pdflatex':
+            if ('T2A' in self.elements['fontenc'] and
+                not self.babel.uses_cyrillic()):
+                self.elements['substitutefont'] = '\\usepackage{substitutefont}'
+                self.elements['sphinxpkgoptions'] += ',cyrnocyr'
+            if 'LGR' in self.elements['fontenc']:
+                self.elements['substitutefont'] = '\\usepackage{substitutefont}'
```

Comment:
Sphinx does not support Greek as main language I think, so it makes sense not to worry about LGR being *last*  font encoding in ``'fontenc'`` key.